argilla-server/src/argilla_server/bulk/records_bulk.py

```diff
@@ -99,10 +100,10 @@ async def _upsert_records_suggestions(
                     try:
                         SuggestionCreateValidator(suggestion_create).validate_for(question.parsed_settings, record)
                         upsert_many_suggestions.append(dict(**suggestion_create.dict(), record_id=record.id))
-                    except ValueError as ex:
+                    except (UnprocessableEntityError, ValueError) as ex:
```

Comment:
Once that everything is refactored I will change these kind of lines to be something like:

```suggestion
                    except UnprocessableEntityError as ex:
```

Reply:
Okay but the raised error shouldn't be UnprocessableEntity?

Reply:
Yes, it's changed to raise `UnprocessableEntityError` in another PR: https://github.com/argilla-io/argilla/pull/4912/files#diff-8b6a4a23212d32e0db897e0fd54b482d2246be5f98c24915ce4f93923f14e8d5R107